modules/electrophysiology_browser/test/TestPlan.md

```diff
@@ -19,6 +19,17 @@
 11. Click each "Download" button (there should be 5). Check: Does the download button work? Does the file that is downloaded have greater than 0kb size? Is a different file downloaded by each button? 
 12. Test Breadcrumb link back to Electrophysiology Browser. [Automated Testing]
 
+### C. Visualization  
+13. Follow the [module README extra installation steps](modules/electrophysiology_browser#installation-requirements-to-use-the-visualization-features) 
```

Comment:
This link isn't right. I think it should be "../README.md#installation-requirements-to-use-the-visualization-features" (haven't tested). It's currently pointing to a directory and the `modules/electrophysiology_browser` part is wrong since it's not in the root of the repo, so it would resolve to `modules/electrophysiology_browser/test/modules/electrophysiology_browser`